.npmrc

```diff
@@ -1 +1,2 @@
-strict-peer-dependencies=false
\ No newline at end of file
+strict-peer-dependencies=false
+registry=https://registry.npmjs.org/
```

Comment:
Isn't this the default value? Any context on why you added this?

Reply:
Sounds good to me 👍🏻 

It is the default value, but as someone who works with different registries, I frequently run into the situation where I switch projects and the previous (or global) registry setting applies to the new project. So it's pretty easy to accidentally add a different registry to the lock file when installing dependencies because you don't get any warning when that happens. This just enforces that default value regardless of the users' settings.

Reply:
Happy to drop it if you don't want it!

Reply:
Makes sense, no problems. Don't mind these kinds of changes, just want some context in history for why it was added if it ever became as issue.